src/time.cpp

```diff
@@ -69,6 +71,28 @@ namespace g3 {
    /// * format string must conform to std::put_time
    /// This is similar to std::put_time(std::localtime(std::time_t*), time_format.c_str());
 
+   namespace {
+      std::string kNanoSecondIdentifier = "%f";
```

Comment:
What about Milli / Micro / Nano formatting? Should I use following formatting?:
- %f - default which is nano
- %f3 - millisec
- %f6 - microsec
- %f9 = %f